style.css

```diff
@@ -89,3 +89,253 @@
 .social li {
   margin: 10px;
 }
+
+/* work section */
+
+.work-section {
+  display: flex;
+  flex-direction: column;
+  background: #e5e5e5;
+}
+
+.card-section {
+  display: flex;
+  flex-direction: column;
+  margin: 100px 24px 50px 24px;
+  background: #fff;
+  padding: 0;
+  justify-content: flex-start;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.profile {
+  margin: 16px 16px 0 16px;
+}
+
+.card-section h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 12px 16px 5px 16px;
+  display: flex;
+  text-align: left;
+  align-items: center;
+  color: #172b4d;
+}
+
+.canopy {
+  display: flex;
+  flex-direction: row;
+  margin: 12px 75px 0 10px;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 0;
+}
+
+.card-section .cano {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 16px;
+}
+
+.canopy li {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 6px;
+  font-family: poppins, sans-serif;
+  color: #7a869a;
+}
+
+.card-section p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 20px;
+  margin: 5px 16px 0 16px;
+  color: #344563;
+}
+
+.tech {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin: 12px 97px 0 6px;
+}
+
+.tech li {
+  list-style: none;
+  margin: 10px;
+  background: #ebebff;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.card-section a {
+  width: 90px;
+  height: 28px;
+  display: flex;
+  align-items: center;
+  justify-self: flex-start;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  color: #6070ff;
+  text-decoration: none;
+  padding: 12px;
+  margin: 16px;
+  font-family: poppins, sans-serif;
+}
+
+.ctn1 a {
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+  color: #7f8cff;
+  margin: 10px 0;
+}
+
+.card-section a:hover {
+  background-color: #c9cdf5;
+  box-shadow: #6070ff;
+}
+
+/* about */
+
+.about {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  margin: 114px 24px;
+  padding: 0;
+}
```

Comment:
- [ ] Please 🙏🏽 add the border-radius in the right corner to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/235266430-6a511e0b-035a-4461-964d-267955511c57.png) | ![image](https://user-images.githubusercontent.com/58642949/235266458-2a2a5ee4-07e2-4e31-8f31-5c878ede7c39.png) |